sources/api/netdog/src/cli/check_net_config.rs

```diff
@@ -0,0 +1,38 @@
+use super::{check_net_config, error, Result};
+use argh::FromArgs;
+use snafu::OptionExt;
+
+#[derive(FromArgs, PartialEq, Debug)]
+#[argh(subcommand, name = "check-net-config")]
+/// Check the network configuration
+pub(crate) struct CheckNetConfigArgs {}
```

Comment:
In my mind, it makes sense to take a path to a network config as an argument here.

Doing so would eliminate the shared logic between this command and `generate-net-config`, which makes sense because here we just need to validate a file, and don't care about it's location.

Reply:
That works for me. I hadn't really leaned into "just for users and developers" and was trying to keep it more consistent with the rest of the usage in `netdog` but if folks are ok with us sharing use cases, I think its a much cleaner interface for the user.